skabase/SKALogger/SKALogger/release.py

```diff
@@ -4,6 +4,8 @@
 #
 #
 #
+# Distributed under the terms of the none license.
```

Comment:
These lines should be removed.

Reply:
This is an outdated change. It's not there in recent code.